runtime/compiler/rpc/J9Client.hpp

```diff
@@ -82,11 +95,58 @@ class J9ClientStream : J9Stream
 
    void shutdown();
 
+   template <typename ...T>
+   void writeError(J9ServerMessageType type, T... args)
+      {
+      _cMsg.set_type(type);
+      if (type == J9ServerMessageType::compilationAbort)
+         _cMsg.mutable_data()->clear_data();
+      else
+         setArgs<T...>(_cMsg.mutable_data(), args...);
+      writeBlocking(_cMsg);
+      }
+
+   VersionCheckStatus getVersionCheckStatus()
+      {
+      return _versionCheckStatus;
+      }
+
+   void setVersionCheckStatus()
+      {
+      _versionCheckStatus = PASSED;
+      }
+
+   static void incrementIncompatibilityCount(OMRPortLibrary *portLibrary)
+      {
+      OMRPORT_ACCESS_FROM_OMRPORT(portLibrary);
+      _incompatibilityCount += 1;
+      _incompatibleStartTime = omrtime_current_time_millis();
+      }
+
+   static uint32_t getIncompatibilityCount(OMRPortLibrary *portLibrary)
```

Comment:
I see that this function is only used in `isServerCompatible()` below. Maybe include its body into `isServerCompatible()` and remove `getCompatibilityCount()`. Also, please add some comments describing the intention of the code.